tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,48 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+from marionette_driver import By, Wait, expected
+
+class BugzillaLite(Base):
+
+    name = 'Bugzilla Lite'
+    _given_username = (By.CSS_SELECTOR, "#login input[type='email']")
+    _given_password = (By.CSS_SELECTOR, "#login input[type='password']")
+    _button_login = (By.CSS_SELECTOR, "#login input[type='submit']")
+    _profile_icon = (By.CSS_SELECTOR, ".profile")
```

Comment:
yea I agree, it should be like that: _profile_icon = (By.CSS_SELECTOR, "a[href='/profile/']")